### PR TITLE
Meta/bump deps

### PR DESCRIPTION
**Pull request by georgehrke**

### CODE REVIEW

File: js/bower.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   "name": "nextcloud-calendar",
   "dependencies": {
-    "angular": "1.6.4",
+    "angular": "1.6.8",
     "angular-bootstrap": "2.5.0",
     "jquery-timepicker": "883bb2cd94",
     "jstzdetect": "https://github.com/georgehrke/jstimezonedetect.git",
     "ical.js": "1.2.2",
-    "fullcalendar": "3.4.0",
+    "fullcalendar": "3.8.0",
```
Comment (tcitworld) [inline js/bower.json]: fullcalendar 3.8.1 was released yesterday

Comment (georgehrke) [reply js/bower.json]: Ok, I will update that.

File: js/app/models/calendarmodel.js [hunk 2]
```diff
@@ -90,7 +90,7 @@ app.factory('Calendar', function($window, Hook, VEventService, TimezoneService,
 
 				return Promise.all(promises).then(() => {
 					callback(vevents);
-					fcAPI.reportEventChange();
+					fcAPI.eventManager.currentPeriod.release();
```
Comment (georgehrke) [inline js/app/models/calendarmodel.js]: This is still quite hacky, but I hope we can get https://github.com/fullcalendar/fullcalendar/issues/1672 soon

File: js/bower.json (hunk 1 above)
Comment (georgehrke) [reply js/bower.json]: done

**Comment by georgehrke** [comment]
Tests passed, let's get this in? :)

**Comment by tcitworld** [comment]
Tested and looks fine.

**Comment by georgehrke** [comment]
Thx for testing :)

**Pull request merged**